src/travel-details-screens/DepartureDetailsScreenComponent.tsx

```diff
@@ -75,6 +80,19 @@ export const DepartureDetailsScreenComponent = ({
   ] = useDepartureData(activeItem, 20, !isFocused);
   const mapData = useMapData(activeItem);
 
+  const realtimeMapEnabled = useRealtimeMapEnabled();
+  const screenReaderEnabled = useIsScreenReaderEnabled();
+
+  const shouldShowLive = isDateInRangeFromNow(activeItem.serviceDate, 12 * 60);
```

Comment:
Estimated calls should have a field `realtime` which maybe can be used for this instead, if I understand correctly what the purpose is. Could you look into that? By using a flag from Entur we might not need any custom logic on this.